KafkaIngester/KafkaIngester.gradle

```diff
@@ -42,3 +42,36 @@ dependencies {
     testRuntimeOnly project(':log-to-slf4j')
     testRuntimeOnly 'org.slf4j:slf4j-simple:1.7.30'
 }
+
+def createDeephavenTestExecTask = {
+    taskName, mainClass -> tasks.create(taskName, JavaExec, { JavaExec task ->
+        new File("$rootDir/tmp/workspace").mkdirs()
+        new File("$rootDir/tmp/logs").mkdirs()
+
+        task.workingDir "$rootDir"
+        task.classpath = sourceSets.test.runtimeClasspath
+        task.main = mainClass
+
+        task.jvmArgs '-DConfiguration.rootFile=dh-tests.prop',
```

Comment:
This whole block is indented weird.

It also has some boilerplate that should maybe come out:
```
                    '-Dlog4j.configuration=log4j.teamcity.xml',
                    '-Ddisable.pdsport.rotate=true',
                    '-Djava.awt.headless=true'
```

Then again, I see that you copied this from elsewhere.